code/modules/shareddream/brainghost.dm

```diff
@@ -20,6 +20,27 @@
 	loc = pick(dream_entries)
 	body = form
 
+	if(client)
+		healths = new /obj/screen()
+		healths.icon = 'icons/mob/screen/midnight.dmi'
+		healths.icon_state = "health0"
+		healths.name = "health"
+		healths.screen_loc = ui_health
+		client.screen |= healths
+
+/mob/living/brain_ghost/LateLogin()
+	..()
+	if(!healths)
+		healths = new /obj/screen()
+		healths.icon = 'icons/mob/screen/midnight.dmi'
+		healths.icon_state = "health0"
+		healths.name = "health"
+		healths.screen_loc = ui_health
+	client.screen |= healths
```

Comment:
This will add it again every login. Also, not keen on this code being duplicated on latelogin. Just the init one should be enough. Does human HUD code do this too?

In addition, why not just add the original mob's health hud item and be done with it?